rest/handler.go

```diff
@@ -755,6 +769,12 @@ func (h *handler) checkPublicAuth(dbCtx *db.DatabaseContext) (err error) {
 			}
 		} else {
 			dbCtx.DbStats.Security().AuthSuccessCount.Add(1)
+
+			if h.isGuest() {
```

Comment:
Can you pass an CB Server admin user on the public port and have it get authorized?